### PR TITLE
Citus nightly pipeline development

### PR DESCRIPTION
**Pull request by gurkanindibay**
- [ ] There was a small bug which was preventing package push into citusdata/tools#173. After its completion tag name should be changed with the new tag

### CODE REVIEW

File: .github/workflows/build-citus-nightlies.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Clone tools branch
+        run: git clone -b pc_invalid_repo_name --depth=1  https://github.com/citusdata/tools.git tools
+
+      - name: Clone all-citus branch
+        run: git clone -b all-citus --depth=1  https://github.com/citusdata/packaging.git packaging
```
Comment (onurctirtir) [inline .github/workflows/build-citus-nightlies.yml]: what about enterprise ?

Comment (gurkanindibay) [reply .github/workflows/build-citus-nightlies.yml]: I can add matrix structure and add enterprise in it but I prefer another PR to make nightly quickly work

Comment (gurkanindibay) [reply .github/workflows/build-citus-nightlies.yml]: @onurctirtir having another pipeline is more accurate way to handle enterprise . I will open another PR for it.
But I need this pipeline to test 10.2.0 release on Github Actions just like we release 10.2.0

Comment (hanefi) [reply .github/workflows/build-citus-nightlies.yml]: Please open an issue to make sure we do not forget about enterprise nightly images

**Review by hanefi — APPROVED** [review]
PR looks like it's ok when I read it. Sadly no easy way to run it.

Please open an issue for a similar cron job for enterprise packages as well.

I think the best way to see if this cron works properly is merging it and checking the outputs in the following days.

**Pull request merged**